feat: add new components

ruchiragoswami opened the pull request:
Add new components : checkbox, radio button 

![newComponent](https://user-images.githubusercontent.com/67211994/154203428-ee0febb3-0711-4086-9356-d2b153bc7218.JPG)

ruchiragoswami committed:
feat: add new components

checkbox, radio button

ruchiragoswami committed:
style: checkbox and radio

kept them left-aligned, user can align these as they want

RakshaPawar108 commented on 1-documentation/doc-inputs/doc-inputs.html:
```diff
@@ -131,6 +131,84 @@ <h2> Input with error </h2>
+        <div class="product-filters">
+            <h3> Rating </h3>
+            <section>
+                <label for="four-stars"> <input type="radio" id="four-stars" name="four-stars" value="four-stars"> 4
```
As you can see in the attached screenshot, I am able to select all radio buttons at once whereas the purpose of radio button is to be able to select only one option at a time. You can fix this issue by giving same `name`  attribute to all the radio buttons among which you want the user to select only one option. Here you can give the `name` as `star-rating` and that will do the job.
![Screenshot (130)](https://user-images.githubusercontent.com/30971358/154205289-334fa909-f294-4fc1-9310-64a994de4471.png)

RakshaPawar108 commented on 1-documentation/doc-inputs/doc-inputs.html:
```diff
@@ -131,6 +131,84 @@ <h2> Input with error </h2>
+        <div class="product-filters">
+            <h3> Sort by </h3>
+            <section>
+                <label for="low-to-high"> <input type="radio" id="low-to-high" name="low-to-high" value="low-to-high">
```
Similarly here you can give `name` attribute as `sort-by-price` to allow user to select only one radio button among the 2.

mohammed-sas reviewed (approved):
other than suggested by raksha your pr is good to merge

ruchiragoswami committed:
fix: Raksha's Review

name attribute in all the radio buttons fixed, was wrongly given as different names.

ruchiragoswami committed:
fix: respective iframes

iframes fixed as per Raksha's review

ruchiragoswami committed:
refactor: comment removed

from css file

RakshaPawar108 reviewed (approved):
Works fine Ruchi. Code looks good to merge. Great work!
